Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1775,6 +1874,210 @@ def get_last_alert_fetched_time(last_run, alert_time_to_fetch):
     return last_alert_fetched_time
 
 
+def list_indicators_command(client: MsClient, args: Dict[str, str]) -> Tuple[str, Optional[Dict], Optional[List]]:
+    """
+
+    Args:
+        client: MsClient
+        args: arguments from CortexSOAR. May include 'indicator_id'
+
+    Returns:
+        human_readable, outputs.
+    """
+    indicators = client.list_indicators(args.get('indicator_id'))
+    limit = int(args.get('limit', 50))
+
+    indicators = indicators[:limit]
+    if indicators:
+        human_readable = tableToMarkdown(
+            'Indicators from Microsoft ATP:',
+            indicators,
+            headers=[
+                'id',
+                'action',
+                'threatType',
+                'severity',
+                'fileName',
+                'fileHashType',
+                'fileHashValue',
+                'domainName',
+                'networkIPv4',
+                'url'
+            ],
+            removeNull=True
+        )
+        return human_readable, {'MicrosoftATP.Indicators(val.id == obj.id)': indicators}, indicators
+    else:
+        return 'No indicators found', None, None
+
+
+def get_future_time(expiration_time) -> str:
```

Comment:
please move this to helper functions section, add type hint for the parameter and add docstring